pages/common/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Execute commands once at a later time make sure the atd (or atrun) is running.
+
+- Execute commands from standard input in 5 minutes (press CTRL+D after entering commands):
+
+`at now + 5 min`
+
+- Execute a command from standard input at 10:00 AM today:
+
+`echo "{{./make_db_backup.sh}}" | at 1000`
+
+- Execute a command from some_file at 9:30 PM Tuesday:
```

Comment:
Yay! The at* commands really do deserve their own page - `at --help` is not particularly, um, _helpful_.

Perhaps here "next Tuesday" would be clearer here?